one_fm/utils.py

```diff
@@ -2303,4 +2303,16 @@ def generate_code():
 
 frappe.whitelist()
 def fetch_employee_signature(user_id):
-    return frappe.get_value("Employee", {"user_id":user_id},["employee_signature"])
\ No newline at end of file
+    return frappe.get_value("Employee", {"user_id":user_id},["employee_signature"])
+
+
+# date utilities
+class DateUtil:
```

Comment:
To get first and last day of month, we can use frappe.utils.get_first_day(date) and  frappe.utils.get_last_day(date).
No need to create a class for this.